Command/UpdateCoreRoutesCommand.php

```diff
@@ -56,10 +56,10 @@ public function execute(InputInterface $input, OutputInterface $output)
 
         foreach ($this->getSites($input) as $site) {
             if ($input->getOption('site') != 'all') {
-                $this->getRoutePageGenerator()->update($site, $output);
+                $this->getRoutePageGenerator()->update($site, $output, (bool) $input->getOption('clean'));
                 $output->writeln('');
             } else {
-                $p = new Process(sprintf('%s sonata:page:update-core-routes --env=%s --site=%s %s', $input->getOption('base-command'), $input->getOption('env'), $site->getId(), $input->getOption('no-debug') ? '--no-debug' : ''));
+                $p = new Process(sprintf('%s sonata:page:update-core-routes --env=%s --site=%s %s %s', $input->getOption('base-command'), $input->getOption('env'), $site->getId(), $input->getOption('no-debug') ? '--no-debug' : '', $input->getOption('clean') ? '--clean' : ''));
```

Comment:
Please 
- linebreak this
- consider using the process builder instead, it should be safer because of escaping.